autogluon/utils/tabular/ml/models/lgb/lgb_model.py

```diff
@@ -277,3 +277,9 @@ def _set_default_searchspace(self):
         for key in self.nondefault_params: # delete all user-specified hyperparams from the default search space
             _ = def_search_space.pop(key, None)
         self.params.update(def_search_space)
+
+    def get_model_feature_importance(self):
+        feature_names = self.model.feature_name()
+        importances = self.model.feature_importance()
```

Comment:
Will these be on similar scale as those computed by the other models using `AbstractModel. compute_permutation_importance()`?

Reply:
ditto